app/views/navbar.scala.html

```diff
@@ -3,12 +3,19 @@
 
 @import views.html.bootstrap._
 
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
In most of the scala.html files we put these stylesheet links near the bottom instead of the top. Could you move this one to stick with that convention?

Reply:
Is at the very bottom or under the html/above the scripts better?

Reply:
above the scripts